parallel-install/pkg/postuninstaller/postuninstaller.go

```diff
@@ -0,0 +1,161 @@
+package postuninstaller
+
+import (
+	"context"
+	"github.com/avast/retry-go"
+	"github.com/kyma-incubator/hydroform/parallel-install/pkg/config"
+	"github.com/kyma-incubator/hydroform/parallel-install/pkg/logger"
+	"github.com/kyma-incubator/hydroform/parallel-install/pkg/preinstaller"
+	"github.com/pkg/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/apimachinery/pkg/labels"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/apimachinery/pkg/selection"
+	"k8s.io/client-go/dynamic"
+)
+
+// Config defines configuration values for the PostUninstaller.
+type Config struct {
+	Log                      logger.Interface        //Logger to be used
+	KubeconfigSource         config.KubeconfigSource //KubeconfigSource to be used
+	InstallationResourcePath string
+}
+
+//go:generate mockery --name PostUninstaller
+
+// PostUninstaller removes leftover resources from k8s cluster, added during Kyma installation.
+type PostUninstaller struct {
+	cfg             Config
+	dynamicClient   dynamic.Interface
+	resourceManager preinstaller.ResourceManager
+	retryOptions    []retry.Option
+}
+
+// Output contains lists of Deleted and NotDeleted resources during PostUninstaller uninstallation.
+type Output struct {
```

Comment:
Isn't that code duplication? there is the same public struct in `preinstaller`

Reply:
In fact we could reuse it, yes. But I will remove it anyways!